src/test/java/com/github/tomakehurst/wiremock/extension/responsetemplating/helpers/ParseJsonHelperTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2011 Thomas Akehurst
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.tomakehurst.wiremock.extension.responsetemplating.helpers;
+
+import com.github.jknack.handlebars.Options;
+import com.github.jknack.handlebars.TagType;
+import com.github.jknack.handlebars.Context;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Arrays;
+import java.util.ArrayList;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+public class ParseJsonHelperTest extends HandlebarsHelperTestBase {
+    private ParseJsonHelper helper;
+
+    @Before
+    public void init() {
+        helper = new ParseJsonHelper();
+    }
+
+    @Test
+    public void parsesASimpleJsonObject() throws Exception {
+        String inputJson = "{\"testKey1\": \"val1\", \"testKey2\": \"val2\"}";
+        Object output = render(inputJson, new Object[]{}, TagType.VAR);
+
+        assertThat(output, instanceOf(Map.class));
+        Map<String, Object> result = (Map<String, Object>) output;
+        assertThat(result, aMapWithSize(2));
+        assertThat(result, hasEntry("testKey1", "val1"));
+        assertThat(result, hasEntry("testKey2", "val2"));
+    }
+
+    @Test
+    public void parsesAJsonObjectContainingArray() throws Exception {
+        String inputJson = "{\"arr\": [\"one\", \"two\", \"three\"]}";
+        Object output = render(inputJson, new Object[]{}, TagType.VAR);
+
+        assertThat(output, instanceOf(Map.class));
+        Map<String, Object> result = (Map<String, Object>) output;
+        assertThat(result, aMapWithSize(1));
+        assertThat(result, hasKey("arr"));
+        assertThat(result.get("arr"), instanceOf(ArrayList.class));
```

Comment:
I suggest checking for a `List` here (and in the subsequent tests) rather than `ArrayList` so that this won't break if for some reason the underlying implementation changes.

Reply:
Thanks 😃, that makes sense, I have updated that, let me know if there is anything else @tomakehurst, I'll have a look at #1545 next otherwise!